backend/src/main/java/heartbeat/service/source/github/GithubService.java

```diff
@@ -24,27 +24,23 @@ public class GithubService {
 	public GithubResponse verifyToken(String githubToken) {
 		String token = "token " + githubToken;
 		try {
-			log.info("[Github] Start to query repository_url by token: https://api.github.com/user/repos");
+			log.info("[Github] Start to query repository_url by token");
 			List<String> githubReposByUser = githubFeignClient.getAllRepos(token)
 				.stream()
 				.map(GithubRepos::getHtml_url)
 				.toList();
-			log.info("[Github] Successfully queried repository_data by token");
-			log.info("[Github] Start to query organization_url by token: https://api.github.com/user/orgs");
+			log.info("[Github] Successfully get repository_url by token, repos: " + githubReposByUser);
 
+			log.info("[Github] Start to query organization_url by token");
 			List<GithubOrganizationsInfo> githubOrganizations = githubFeignClient.getGithubOrganizationsInfo(token);
-			log.info("[Github] Successfully queried organization_data by token");
+			log.info("[Github] Successfully get organizations by token, organizations: "
+					+ githubOrganizations.stream().map(GithubOrganizationsInfo::getLogin).toList());
```

Comment:
Whether to consider not performing logical operations in log printing.

Reply:
done